src/Microsoft.AspNetCore.Mvc.Formatters.Xml/XmlDataContractSerializerInputFormatter.cs

```diff
@@ -59,6 +62,27 @@ public XmlDataContractSerializerInputFormatter(bool suppressInputFormatterBuffer
             WrapperProviderFactories.Add(new SerializableErrorWrapperProviderFactory());
         }
 
+        /// <summary>
+        /// Initializes a new instance of DataContractSerializerInputFormatter
```

Comment:
Use `<see cref="DataContractSerializerInputFormatter"/>` (or `<see cref="XmlSerializerInputFormatter"/>`) and end comments with periods.